src/components/ExpandableSection.tsx

```diff
@@ -1,22 +1,35 @@
-import PropTypes from 'prop-types';
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, FunctionComponent } from 'react';
 import Collapse from './Collapse';
 import ClickableContainer from './ClickableContainer';
 import Icon from './Icon';
 
-const ExpandableSection = (props) => {
-  const [open, setOpen] = useState(props.open);
+interface ExpandableSectionProps {
+  children?: React.ReactNode;
+  className?: string;
+  onToggle: (open: boolean) => void;
```

Comment:
`ExpandableSection.defaultProps` declares a default `onToggle` prop:
https://github.com/appfolio/react-gears/blob/1adbf65196bce045bc1596a602db6a4e3998a849/src/components/ExpandableSection.tsx#L54-L57
and there are several tests in `ExpandableSection.spec.js` that do not pass the `onToggle` prop:
https://github.com/appfolio/react-gears/blob/1adbf65196bce045bc1596a602db6a4e3998a849/test/components/ExpandableSection.spec.js#L7-L61
so it seems that this `ExpandableSectionProps` interface should define `onToggle` as an optional prop.

Reply:
This is fixed in https://github.com/appfolio/react-gears/pull/814